Kickstarter-iOS/Views/CommentComposerView.swift

```diff
@@ -119,6 +119,17 @@ final class CommentComposerView: UIView {
   }
 }
 
+// MARK: - Styles
+
+private let rootStackViewStyle: StackViewStyle = { stackView in
```

Comment:
May be duplicated from [CommentCellStyles.swift](https://github.com/kickstarter/ios-oss/pull/1480/commits/1b3df30485d07ec2aaad345fa68d0eb1f958ee95#diff-d5a4fea3497c0722b485362398bc4c619a47f187f5a9c47850c237c37ed5638cR5-R12)

Reply:
That's why I made it `private`.

Reply:
Yep all the styles should be private to their views and VC's, if they're shared then we should move them to a shared location 👍